ern-container-gen/src/reactNativeBundleAndroid.ts

```diff
@@ -3,27 +3,26 @@ import fs from 'fs'
 import path from 'path'
 
 export async function reactNativeBundleAndroid({
+  bundleOutput,
   dev,
   outDir,
   sourceMapOutput,
   cwd,
 }: {
+  bundleOutput?: string
   dev?: boolean
   outDir: string
   sourceMapOutput?: string
   cwd?: string
 }): Promise<BundlingResult> {
   cwd = cwd || process.cwd()
   const libSrcMainPath = path.join(outDir, 'lib', 'src', 'main')
-  const bundleOutput = path.join(
-    libSrcMainPath,
-    'assets',
-    'index.android.bundle'
-  )
+  bundleOutput =
+    bundleOutput || path.join(libSrcMainPath, 'assets', 'index.android.bundle')
   const assetsDest = path.join(libSrcMainPath, 'res')
-  if (fs.existsSync(assetsDest)) {
+  /* if (fs.existsSync(assetsDest)) {
```

Comment:
Not sure, but if this is not needed anymore, remove it? ;)

Reply:
Oh that's a good catch ! :+1: It's actually needed, was commented out for some purpose during development of this feature.